warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/MetadataTableSplits.java

```diff
@@ -130,7 +130,10 @@ private Path createTempFile(FileSystem fs) throws IOException {
         Path tmpSplitsFile = null;
         try {
             do {
-                tmpSplitsFile = new Path(this.splitsPath.getParent(), SPLITS_CACHE_FILE + "." + count);
+                Path parentDirectory = this.splitsPath.getParent();
+                String fileName = SPLITS_CACHE_FILE + "." + count;
+                log.info("Attempting to create " + fileName + " under " + parentDirectory.toString());
```

Comment:
is INFO the right level for the new logging calls? They look more like DEBUG to me.

Reply:
Most of the info log statements would get written during calls to update.  The current design has that happen with the generateSplitsCache.sh script which currently outputs a lot but has very little useful output.  I believe the total number of info logs we're adding to its output is under 10 log entries per run.  There would be one additional info log message per ingest job.

Reply:
I just talked with the author. If what you want is to be able to go back to see 'what happened' after a failure (without running things again with a new log level), I'm fine with the INFO log level for now.